lib/sizzle.patched.js

```diff
@@ -1488,6 +1491,20 @@ Expr = Sizzle.selectors = {
 
 		"contains": markFunction(function( text ) {
```

Comment:
Do you want to reassign the value of `Sizzle.selectors.pseudos["contains"]` somewhere else? Doesn't it require having duplicate codes? Also we need a referrence for `markFunction`.